replit_river/session.py

```diff
@@ -203,9 +203,14 @@ async def _handle_messages_from_ws(
                             )
                         await self._add_msg_to_stream(msg, stream)
                     else:
-                        stream = await self._open_stream_and_call_handler(
-                            msg, stream, tg
-                        )
+                        # TODO(dstewart) This looks like it opens a new call to handler
+                        #                on ever ws message, instead of demuxing and
+                        #                routing.
```

Comment:
The TODO is because we don't attempt to clean up the previous stream -- this class is (seems to be?) intended for a single stream, so if we attempt to open another one it should be an error (again, at least to my reading).